frontend/src/app/oauth-service.ts

```diff
@@ -0,0 +1,3 @@
+export class OauthService {
```

Comment:
Little awkward to have this here since it's currently deadcode. But should be used in a later commit.